app/models/event.rb

```diff
@@ -8,4 +8,6 @@ class Event < ApplicationRecord
             presence: true
   validates :name,
             presence: true
+
+  scope :upcoming, -> { where('date >= ?', Time.zone.today) }
```

Comment:
Because our events don't have times, if the event is for today, it will be skipped with Time.current because it is equivalent to 00:00:00